tests/post/steps/test_logging.py

```diff
@@ -194,6 +194,28 @@ def _check_log_line_exists():
     )
 
 
+@then("we can retrieve cluster alerts from Loki API")
+def retrieve_cluster_alerts_from_loki(k8s_client):
+    query = {
+        "query": '{app="metalk8s-alert-logger"}',
+    }
+
+    def _check_log_line_exists():
+        response = query_loki_api(k8s_client, query, route="query_range")
+        try:
+            result_data = response[0]["data"]["result"][0]["values"]
+        except (IndexError, KeyError):
+            result_data = []
+        assert result_data, "No cluster alerts found in Loki"
```

Comment:
Don't we want to add an extra check to verify we can see the Watchdog alert there, as suggested in the design? This way, we can also ensure the format of such logs is stable over time.

Reply:
Oh, yes we could, I didn't make it specific to Watchdog just in case it would change at some point, to avoid having to change the test and because the purpose was only to ensure the alert history is working fine.
But, you're right, this way we could also ensure we log valid JSON objects, with specific fields, etc.
I will change it.